obspy/core/util/base.py

```diff
@@ -382,7 +382,8 @@ def get_proj_version(raw_string=False):
     # proj4 is a c library, prproj wraps this.  proj_version is an attribute
     # of the Proj class that is only set when the projection is made. Make
     # a dummy projection and get the version
-    version_string = str(Proj(proj='utm', zone=10, ellps='WGS84').proj_version)
+    _proj = Proj(proj='utm', zone=10, ellps='WGS84')
+    version_string = str(getattr(_proj, 'proj_version_str', 'proj_version'))
```

Comment:
I'm pretty sure this is not doing what you want it to do.

I would just do a `if hasattr(...):` if/else

Reply:
Yup, you are right. The third argument is just a default value.